.circleci/config.yml

```diff
@@ -79,6 +83,9 @@ jobs:
             - codecov/upload:
                 file: './coverage/clover.xml'
                 token: 74ebd8f2-0b51-4873-972a-82c2b3509b37
+            - codecov/upload:
+                file: './cypress-coverage/clover.xml'
+                token: 74ebd8f2-0b51-4873-972a-82c2b3509b37
```

Comment:
With two upload steps here is codecov smart enough to know that these are different files and not just override the first one with the second?